v0/destinations/posthog/transform.js

```diff
@@ -0,0 +1,105 @@
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  constructPayload,
+  defaultPostRequestConfig
+} = require("../../util");
+const { Message } = require("../../util/enum/message");
+
+// Logic To match destination Property key that is in Rudder Stack Properties Object.
+const generatePropertyDefination = message => {
```

Comment:
Could we incorporate all the fields to the individual config JSONs instead of putting it here separately

Reply:
Fixed